lib/components/data/plots/CWMSPlot.jsx

```diff
@@ -182,13 +216,13 @@ export default function CWMSPlot({
   return (
     <div
       className={gwMerge("gww-h-full gww-w-full", className)}
-      style={{ height: plotHeight }}
+      style={{ height: layoutParams.height }}
```

Comment:
This throws if `layoutParams` isn't defined.  Should probably set some sensible defaults for size.